tests/test.sh

```diff
@@ -18,7 +35,8 @@ RAM_ID=5 # Location of RAM usage value in kubectl top's output
 
 TEST_NAME=$1
 SPECIFIC_TEST_NAME=$(cut -d "/" -f 2 <<< ${TEST_NAME})
-RELEASE=network-stresser-test-${SPECIFIC_TEST_NAME}
+RELEASE_GEN=network-stresser-test-${SPECIFIC_TEST_NAME}-gen
```

Comment:
Now I also understand why you created gen and rec ... I think this is replication of a lot of code, maybe just replicated the output somehow .... also use names like receiver and generator ... no need for those short names

Reply:
I beg to differ. I do not see any piece of code that is replicated among stable_gen and stable_rec, for each of the two directories contains only the relevant files for the generators or the receivers respectively. In order to allow for a separate initialization of each of the two components their helm charts must reside in two different directories or else they will be deployed together.